<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>DICOM</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>677</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <property name="margin">
            <number>6</number>
           </property>
           <item>
            <widget class="QCheckBox" name="overlays_checkBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Load overlays</string>
             </property>
             <property name="text">
              <string>Load overlays</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="mosaic_checkBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Enable Siemens Mosaic</string>
             </property>
             <property name="text">
              <string>Process Siemens Mosaic / UIH Grid</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Advanced</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <item>
               <widget class="QCheckBox" name="clean_unused_checkBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clean unused bits if &lt;span style=&quot; font-style:italic;&quot;&gt;BitsAllocated &amp;gt; BitsStored. &lt;/span&gt;Slower.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Clean unused bits</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="pet_no_level_checkBox">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Don't apply Level/Window to PET</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="time_s__checkBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="mouseTracking">
                 <bool>false</bool>
                </property>
                <property name="toolTip">
                 <string>Default: ms</string>
                </property>
                <property name="text">
                 <string>FrameTime/FrameTimeVector attributes time in seconds</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="rescale_checkBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="mouseTracking">
                 <bool>false</bool>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set to TRUE! Only to experiment with data set.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="statusTip">
                 <string/>
                </property>
                <property name="text">
                 <string>Apply Rescale Intercept/Slope (must always be TRUE)</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Application</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>677</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="margin">
            <number>6</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Font size in points</string>
               </property>
               <property name="textFormat">
                <enum>Qt::PlainText</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="pt_doubleSpinBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="specialValueText">
                <string/>
               </property>
               <property name="suffix">
                <string> pt</string>
               </property>
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.200000000000000</double>
               </property>
               <property name="value">
                <double>12.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="title">
       <string>OpenGL</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>677</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="margin">
            <number>6</number>
           </property>
           <item>
            <widget class="QGroupBox" name="textureoptions_groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>OpenGL texture</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QRadioButton" name="original_radioButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="mouseTracking">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Try original dimensions, try to downsample if required.</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="sizeConstraint">
                   <enum>QLayout::SetDefaultConstraint</enum>
                  </property>
                  <item>
                   <widget class="QRadioButton" name="resample_radioButton">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="mouseTracking">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Maximum size</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line_5">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="x_label">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>X:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="x_comboBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::ClickFocus</enum>
                    </property>
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="y_label">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Y:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="y_comboBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::ClickFocus</enum>
                    </property>
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="filtering_groupBox">
                <property name="toolTip">
                 <string>Applicable to uniform volumes</string>
                </property>
                <property name="title">
                 <string>Filtering</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <property name="margin">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="f_no_radioButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="mouseTracking">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>No</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="f_bilinear_radioButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="mouseTracking">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Bilinear</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="maxvbos_checkBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Max OpenGL VBOs 65535</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>520</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="reload_pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../alizams.qrc">
       <normaloff>:/bitmaps/reload.svg</normaloff>:/bitmaps/reload.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>scrollArea_3</tabstop>
  <tabstop>overlays_checkBox</tabstop>
  <tabstop>mosaic_checkBox</tabstop>
  <tabstop>clean_unused_checkBox</tabstop>
  <tabstop>pet_no_level_checkBox</tabstop>
  <tabstop>time_s__checkBox</tabstop>
  <tabstop>rescale_checkBox</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>pt_doubleSpinBox</tabstop>
  <tabstop>scrollArea_2</tabstop>
  <tabstop>textureoptions_groupBox</tabstop>
  <tabstop>original_radioButton</tabstop>
  <tabstop>resample_radioButton</tabstop>
  <tabstop>f_no_radioButton</tabstop>
  <tabstop>f_bilinear_radioButton</tabstop>
  <tabstop>maxvbos_checkBox</tabstop>
  <tabstop>reload_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../alizams.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>resample_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>x_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resample_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>x_comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resample_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>y_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resample_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>y_comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
